### ISSUE TITLE
Build a clean user interface for Url shortner 

### ISSUE THREAD

**Issue by pranavbharadwaj007** [Hacktoberfest-SCE/Lets-Build#6]
Project : https://github.com/Hacktoberfest-SCE/Lets-Build/blob/main/URL%20shortener/main.py

**Comment by neeraj15022001** [comment]
do you need this one to be created in tinker kit ? I mean is it desktop or web app ?

**Comment by pranavbharadwaj007** [comment]
no, i already created the project u just need to add more features
i have used Tkintner https://github.com/Hacktoberfest-SCE/Lets-Build/blob/main/URL%20shortener/main.py
